Component generator crashes

as reported by Michele Pinto on Polymer slack

```
~/workspace/playground/apollo-elements-app % npm init @apollo-elements --version
6.14.8
~/workspace/playground/apollo-elements-app % npm init @apollo-elements component
npx: installed 44 in 5.057s
internal/modules/cjs/loader.js:985
  throw err;
  ^
Error: Cannot find module 'mkdirp'
Require stack:
- .npm/_npx/19411/lib/node_modules/@apollo-elements/create/component.js
- .npm/_npx/19411/lib/node_modules/@apollo-elements/create/index.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:982:15)
    at Function.Module._load (internal/modules/cjs/loader.js:864:27)
    at Module.require (internal/modules/cjs/loader.js:1044:19)
    at require (internal/modules/cjs/helpers.js:77:18)
    at Object.<anonymous> (/Users/michelepinto/.npm/_npx/19411/lib/node_modules/@apollo-elements/create/component.js:7:42)
    at Module._compile (internal/modules/cjs/loader.js:1158:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1178:10)
    at Module.load (internal/modules/cjs/loader.js:1002:32)
    at Function.Module._load (internal/modules/cjs/loader.js:901:14)
    at Module.require (internal/modules/cjs/loader.js:1044:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '.npm/_npx/19411/lib/node_modules/@apollo-elements/create/component.js',
    '.npm/_npx/19411/lib/node_modules/@apollo-elements/create/index.js'
  ]
}
```

--- a/package.json
+++ b/package.json
@@ -67,6 +67,8 @@
     "@types/chai": "^4.2.12",
     "@types/graphql": "^14.5.0",
     "@types/jsdom": "^16.2.4",
+    "@types/mkdirp": "^1.0.1",
+    "@types/ncp": "^2.0.4",
     "@types/prompts": "^2.0.9",
     "@typescript-eslint/parser": "^4.1.1",
     "@web/dev-server": "^0.0.10",

--- a/packages/create/package.json
+++ b/packages/create/package.json
@@ -31,6 +31,7 @@
   "dependencies": {
     "case": "^1.6.3",
     "execa": "^4.0.3",
+    "mkdirp": "^1.0.4",
     "ncp": "^2.0.0",
     "prompts": "^2.3.2",
     "tslib": "^2.0.1",

